SECURITY.md

```diff
@@ -1,12 +1,12 @@
 # Security Policy
 
-The Indico Team takes security very seriously. We greatly value the efforts of those who help us maintain the
-security and integrity of the project. If you discover a vulnerability, we ask that you report it responsibly
-so that we can fix it promptly.
+The Indico Team takes security very seriously. We greatly value the efforts of those who help us
```

Comment:
Interesting. Did you use some automated tool to do this? Just wondering what the standard should be, or whether it's a matter of personal taste.